lib/Cake/Utility/Security.php

```diff
@@ -63,7 +63,7 @@ public static function inactiveMins() {
  * @return string Hash
  */
 	public static function generateAuthKey() {
-		return Security::hash(CakeText::uuid());
+		return Security::hash(openssl_random_pseudo_bytes(32));
```

Comment:
I thought we deprecated this method a long long time ago.. Because we don't explicitly require ext/openssl  we'll need to check that the module exists first.